client/packages/invoices/src/Returns/modals/CustomerReturn/useDraftCustomerReturnLines.ts

```diff
@@ -120,6 +121,7 @@ export const useDraftCustomerReturnLines = ({
         customerId,
         outboundShipmentId,
         customerReturnLines,
+        status: InsertCustomerReturnStatusInput.Shipped,
```

Comment:
This hook is only used when creating returns from shipments, so we can use shipped for this case.

Reply:
This should be `Verified`